android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -305,6 +308,7 @@ public abstract class BraveActivity extends ChromeActivity
     private static final List<String> sYandexRegions =
             Arrays.asList("AM", "AZ", "BY", "KG", "KZ", "MD", "RU", "TJ", "TM", "UZ");
 
+    private static final int PIP_UPDATE_DELAY_MS = 500;
```

Comment:
Where this number comes from? Do they have the same delay in the upstream?

Reply:
Yes, correct. I've added a comment below that also references the same Java class so the code can be checked further if required.

Reply:
More details in the PR description as well.